src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -277,6 +287,15 @@ def register_source_uri_arguments(scope):
 register_cli_argument('storage blob copy', 'blob_name', blob_name_type, options_list=('--destination-blob', '-b'), help='Name of the destination blob. If the exists, it will be overwritten.')
 register_cli_argument('storage blob copy', 'source_lease_id', arg_group='Copy Source')
 
+# BLOB INCREMENTAL COPY PARAMETERS
+register_blob_source_uri_arguments('storage blob incremental-copy start')
```

Comment:
You can find information about incremental blob copy here: https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/incremental-copy-blob

The batch copy is a subcommand of copy. And we, storage team and me, decide to single the out incremental copy as a separate command because it applies to narrower options and it will be confusing to user if we jam all the parameters and combination in one command.